tests/test_graphic_walker.py

```diff
@@ -81,6 +81,13 @@ def test_process_parameter_change_with_appearance(data):
     result = gwalker._process_param_change(params)
     assert result["appearance"] == appearance
 
+def test_process_parameter_change_resetting_server_computationt(data):
```

Comment:
There is another bug that I could not figure out how to solve. This is my attempt to add a test for this.

When you change from `server_computation=True` to `server_computation=False` the `object` data should be sent to the client in the same message as the `server_computation` value. If not an exception is raised in the client. You can see it below:

https://github.com/user-attachments/assets/2e305906-a535-4156-8f84-118e61bfcc77

I really tried to find a fix - but had to give up. I get 

```bash
  File "/home/jovyan/repos/private/panel-graphic-walker/.venv/lib/python3.11/site-packages/pandas/core/generic.py", line 1577, in __nonzero__
    raise ValueError(
ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```

with the solution I tried. Hope you can help?